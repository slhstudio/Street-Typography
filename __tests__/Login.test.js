import React from 'react';
import LogInComponent from '../client/components/Login';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';

describe('<LogIn/>', () => {
  it ('renders and displays properly', () => {
    const wrapper = shallow(<LogInComponent/>);
    expect(toJSON(wrapper)).toMatchSnapshot();
    // expect(wrapper.find('.loginbox').exists());
    // expect(wrapper.find('p').exists()).toBe(true);
    // expect(wrapper.find('a').exists()).toBe(true);
    // expect(wrapper.find('button').exists()).toBe(true);
    // expect(wrapper.find('button').text()).toBe('Google');
  });
});