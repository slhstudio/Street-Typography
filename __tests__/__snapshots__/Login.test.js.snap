// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LogIn/> renders and displays properly 1`] = `
<div
  className="loginbox"
>
  <p>
     Log in or Sign in with
  </p>
  <a
    href="/auth/google"
  >
    <button>
      Google
    </button>
  </a>
</div>
`;
